openapi/docs/DefaultAPI.md

```
@@ -13,6 +16,204 @@ Method | HTTP request | Description
+```go
+package main
+
+import (
+	"context"
+	"fmt"
+	"os"
+	openapiclient "github.com/blinklabs-io/bursa/openapi"
+)
+
+func main() {
+	request := *openapiclient.NewApiScriptAddressRequest("Network_example", map[string]map[string]interface{}{"key": map[string]interface{}(123)}) // ApiScriptAddressRequest | Script Address Request
```

markdownlint-cli2 (0.18.1)

Hard tabs
Column: 1

(MD010, no-hard-tabs)